src/main/java/seedu/dictionote/ui/NoteContentPanel.java

```diff
@@ -14,7 +14,7 @@
 import seedu.dictionote.model.note.Note;
 
 /**
- * An UI component that displays information of a {@code DisplayableContent}.
+ * An UI component that displays and edit of a {@code Note} information.
```

Comment:
* Shouldn't it be *"A UI"* instead of *"An UI"*?
* *"...that displays **and edit of a**..."* sounds a bit ambiguous. Should it be *"and edits a"* instead?

Reply:
fixed